init.go

```diff
@@ -44,6 +44,7 @@ func init() {
 	flag.BoolVar(&apply, "apply", false, "apply the plan directly")
 	flag.BoolVar(&debug, "debug", false, "show the execution logs")
 	flag.BoolVar(&dryRun, "dry-run", false, "apply the dry-run option for helm commands.")
+	flag.Var(&target, "target", "limit execution to specific app.")
```

Comment:
Since with the flag you can define one specific app, I went for `app`, not `apps`. You can still use multiple `-target` flags of course, but `app or apps` might confuse some people on how to define multiple application within one flag: "Is it like with comma, spaces or what?". Does this explanation convince you? :-)

Reply:
@mkubaczyk thanks for the PR :) ..  perhaps a good clarification would be to add a short documentation in the how_to folder under the Misc section explaining how to target specific app/apps in an execution. Something like this https://github.com/Praqma/helmsman/blob/master/docs/how_to/merge_desired_state_files.md

Reply:
otherwise, this is ready to merge :)

Reply:
I will do this soon and let you know, thanks :-)

Reply:
@sami-alajrami please check https://github.com/Praqma/helmsman/pull/237/commits/736b106aff39aab749350ee7619dc99350ac3975. I've defined v1.9.0 for this doc, correct me if this assumption is wrong. :-)